core/src/main/java/org/apache/calcite/sql/ddl/SqlDdlNodes.java

```diff
@@ -61,6 +61,14 @@ public static SqlCreateTable createTable(SqlParserPos pos, boolean replace,
         query);
   }
 
+  /** Creates a CREATE TABLE LIKE. */
+  public static SqlCreateTableLike createTableLike(SqlParserPos pos, boolean replace,
```

Comment:
compiler error for SqlCreateTableLike
we can add core/src/main/java/org/apache/calcite/sql/ddl/SqlCreateTableLike.java